fiftyone/core/annotation/utils.py

```
@@ -0,0 +1,207 @@
+        foac.FIELD_TYPE_TO_TYPES[fof.ListField]
+        if is_list
+        else foac.FIELD_TYPE_TO_TYPES
+    )
+
+    if field_type not in _types:
+        raise ValueError(f"field '{field}' is not supported")
+
+    return _types[field_type]
+
+
+def _is_supported_label(field, media_type, require_app_support):
```

Ruff (0.14.8)

Missing return type annotation for private function `_is_supported_label`

(ANN202)